_posts/2021/2021-06-28-calculate-the-standard-deviation-of-a-ruby-array.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Calculate the standard deviation of a Ruby array"
+description: "A lesson in why we use native Ruby methods"
+layout: article
+category: ruby
+image:
+  base: '2020/calculate-a-mean-average-from-a-ruby-array'
+  alt: "Black and White Texas Instruments calculator"
+  credit:  Ray Reyes
+  source: "https://unsplash.com/photos/3xwrg7Vv6Ts"
+
+---
+
+Standard deviation is a measure of the amount of variance of group of values. A high value indicates a wide spread of values around the mean, a low value indicates tight clustering of values.
+
+The calculation is used to understand and give context to the mean value of a series of data, [wikipedia has some good examples of real-world applications](https://en.wikipedia.org/wiki/Standard_deviation#Application_examples).
```

Comment:
Full stop before 'Wikipedia'.

I dunno: "understand and give context too..." sounds wishy washy.

Reply:
```suggestion
The calculation gives additional context to the range of values around the mean value of a series of data. [Wikipedia has some good examples of real-world applications](https://en.wikipedia.org/wiki/Standard_deviation#Application_examples).
```